Components/grid/grid.css

```diff
@@ -0,0 +1,43 @@
+.grid-50-50-format{
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    gap: 1rem;
+    box-shadow: var(--box-shadow-light);
+}
+.grid-70-30-format{
+    display: grid;
+    grid-template-columns:0.7fr 0.3fr;
+    gap: 1rem;
+    box-shadow: var(--box-shadow-light);
+}
+
+.grid-30-70-format{
+    display: grid;
+    grid-template-columns:0.3fr 0.7fr;
+    gap: 1rem;
+    box-shadow: var(--box-shadow-light);
+}
```

Comment:
Everything looks good, though if you want you can combine these three selectors using commas and write the display, gap, and box-shadow property only once.

Reply:
Yes kshitij, thanks for that!